src/tests/Dashboard/reducer/conceptReducer.test.js

```diff
@@ -225,7 +246,29 @@ describe('Test suite for concepts reducer', () => {
         prePopulated: true,
       },
     ];
-    const newState = reducer(state, { type: UN_POPULATE_THIS_ANSWER, payload });
-    expect(newState.selectedAnswers).toEqual(expected);
+    const newState = reducer(state, {
+      type: UN_POPULATE_THIS_ANSWER,
+      payload
+    });
+    expect(newState.selectedAnswers)
+      .toEqual(expected);
+  });
+
+  it('should add the selected sets to redux state', () => {
+    const payload = { answer: { frontEndUniqueKey: 'intialKey' }, uniqueKey: 'intialKey' };
+    const newState = reducer(initialState, { type: ADD_SELECTED_SETS, payload });
+    expect(newState.selectedSets).toEqual([{ frontEndUniqueKey: 'intialKey' }]);
+  });
+
+  it('should add new set row', () => {
+    const payload = { frontEndUniqueKey: 'newIntialKey' };
+    const newState = reducer(initialState, { type: ADD_NEW_SET_ROW, payload });
+    expect(newState.selectedSets).toEqual([{ frontEndUniqueKey: 'intialKey' }, { frontEndUniqueKey: 'newIntialKey' }]);
```

Comment:
How do you tell that it was not equal to this even before the add new set row operation?